demisto_sdk/commands/content_graph/objects/integration.py

```diff
@@ -96,6 +96,14 @@ def to_raw_dict(self) -> Dict:
         remove_nulls_from_dictionary(command)
         return command
 
+    def diff_outputs_context_path(self, other) -> Set[Optional[str]]:
+        """
+        This method returns the context path diff between two commands objects.
+        """
+        outputs_set = {output.contextPath for output in self.outputs}
+        outputs_other_set = {output.contextPath for output in other.outputs}
+        return outputs_set.difference(outputs_other_set)
```

Comment:
Since this diff is only useful in the specific validation we're working on here, let's move it to the validation file (rather than in the integration object)